src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs

```diff
@@ -59,6 +76,9 @@ public async Task ServerAsyncAuthenticate_EachSupportedProtocol_Success(SslProto
                 });
 
             Assert.NotNull(e);
+            // FIXME: Mono sometimes throws IOException instead of AuthenticationException.
+            if (PlatformDetection.IsMono && e is IOException)
```

Comment:
Instead of altering test you should just disable it with tracking issue

Reply:
I think the test is still viable to have - and it's purpose is really to ensure we're throwing _some_ kind of exception; it doesn't really matter too much _which kind of_ exception is thrown as long as it's throwing _something_.

Reply:
Looks like it might also throw `TimeoutException` in some scenarios.  Maybe we should make it just check for "has some exception being thrown"?

Reply:
ok, let's skip `IsAssignableFrom` with FIXME